smoroz/03_HTML/form.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8"><title>Form</title>
+    </head>
+    <body>
+        <table width=700px>
+            <tr>
+                <td valign="top">
+                    <form method="get" name="simple" action="http://sc.it-academy.by/fe/form.php">
+                        <b>Простая форма поиска в вариантами:</b>
+                        <fieldset>
+                            <legend>Cтрока поиска</legend>
+                            <input type="text" id="tetx" maxlength="10" size="10">
+                            <input type="submit" id="submit" value="Искать"><br>
+                            <input type="radio" id="radio_group" name="radio_1" value="value_01"> <label for="radio_group">на сайте</label>
+                            <input type="radio" id="radio_group" name="radio_1" value="value_02" checked><label for="radio_group">в google</label><br>
+                            <i>Длина запроса не более 10 символов</i>
+                        </fieldset>
+                    </form>
+                </td>
+                <td>
+                    <form method="get" name="simple_focus" action="http://sc.it-academy.by/fe/form.php">
+                        <b>Простая форма поиска в вариантами:</b>
+                        <fieldset>
+                            <legend>Cтрока поиска</legend>
+                            <input type="text" id="tetx" size="10" autofocus required><span><sup>*</sup></span>
+                            <input type="submit" id="submit" value="Искать"><br>
+                            <input type="radio" id="radio_group" name="radio_2" value="value_01"><label for="radio_group">на сайте</label>
+                            <input type="radio" id="radio_group" name="radio_2" value="value_02" checked><label for="radio_group">в google</label><br>
+                            <i>*-обязательное поле</i><br>
+                        </fieldset>
+                    </form>
+                </td>
+            <tr>
+                <td valign="top">
+                    <form method="post" name="authority" action="http://sc.it-academy.by/fe/form.php">
+                        <b>Простая форма авторизации:</b>
+                        <fieldset>
+                            <legend>Авторизуйтесь</legend>
+                            <input type="text" name="login"><span><sup>*</sup></span><label for="login">Логин</label><br>
```

Comment:
attribute for needs to connect to the attribute id
it means input should has value of attribute id same as value attribute for
I spend a lot of time to explain that on our last  lesson